src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -237,9 +233,36 @@ const NetworkModificationNodeEditor = ({ selectedNode }) => {
         return dialogs[editDialogOpen].dialog();
     };
 
+    const commit = useCallback(
+        ({ source, destination }) => {
+            if (destination === null) return;
```

Comment:
To avoid useless requests :
if (destination === null || source.index === destination.index) return;